Spoon Master/Controller/HomeViewController.swift

```diff
@@ -16,6 +16,10 @@ final class HomeViewController: UIViewController {
     
     private let repositories = Repositories(api: .share)
     
+    var recipes1: [DataCell] = []
+    var recipes2: [DataCell] = []
+    var products: [DataCell] = []
```

Comment:
```suggestion
    var popularFoods = [DataCell]()
    var popularRecipes = [DataCell]()
    var popularProducts = [DataCell]()
```

Đặt tên rõ ràng vào để sau nếu sửa thì dễ phát hiện hơn, đặt tên đúng theo tableView hiển thị nha